src/Orchard.Web/Modules/Orchard.DynamicForms/Elements/PlaceholderableFormElement.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Orchard.Layouts.Helpers;
+
+namespace Orchard.DynamicForms.Elements {
+    public abstract class PlaceholderableFormElement : LabeledFormElement {
```

Comment:
Thanks @sfmskywalker, I'm agree on both names and see them as appropriate, but let's wait for other suggestions to make the final decision